codepropertygraph/src/main/scala/io/shiftleft/passes/ParallelIteratorExecutor.scala

```diff
@@ -1,10 +1,9 @@
 package io.shiftleft.passes
 
-import scala.concurrent.ExecutionContext.Implicits.global
 import scala.concurrent.duration.Duration
-import scala.concurrent.{Await, Future}
+import scala.concurrent.{Await, ExecutionContext, Future}
 
-class ParallelIteratorExecutor[T](iterator: Iterator[T]) {
+class ParallelIteratorExecutor[T](iterator: Iterator[T])(implicit val executionContext: ExecutionContext) {
```

Comment:
Nitpick: IMO `implicit executionContext`, namely without `val`, would be more accurate as we don't expect anything to access this value